guide/english/algorithms/greatest-common-divisor-direct-analysis/index.md

```diff
@@ -7,6 +7,49 @@ This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/
 
 <a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
 
+## Algorithm Explanation
+
+This algorithm takes any two numbers and finds the greatest common divisor. To do this, it reads in any two numbers (a & b) and makes sure that a is greater than b. It then divides a by b and calculates the remainder. Once it finds this, the value of b becomes a, and the value of the remainder overwrites b. The program repeats itself until b = 0. Once this happens, the value of a will be the greatest common divisor of the two numbers. 
+
+#### Java for Greatest Common Divisor Direct Analysis
+
+import java.util.Scanner;
```

Comment:
add code markup here, insert the code you are trying to add inside ``` ```
for example

like this

```java 
class GCD {
```
Check out GitHub markdown style guides for more information or feel free to ask